client/elements/suttaplex/sc-parallel-item.html

```diff
@@ -258,8 +267,51 @@
                   return item.acronym ? item.acronym : item.uid;
               } else return '';
           }
+
+          _getAcronymTitle(item,localize) {
+              let sc_acronym_title = localize('suttaCentralID');
+              if (item.acronym) {
+                const alt_number = item.acronym.split('//')[1];
+                if (alt_number) {
+                  let book = '';
+                  (alt_number[0] === 'T') ? book = 'Taishō' : book = 'PTS';
+                  sc_acronym_title += `\n${localize('alternateText', 'book', book)} ${alt_number}`;
+                }
+              }
+              return sc_acronym_title;
+          }
+
+          _getParagraphRange(rootId) {
+            const ids = rootId.split(/#(.*)/)[1];
+            if(!ids) {
+                return '';
+            }
+            if(ids.includes('-')) {
+              const [id1, id2] = ids.split('-#');
+              return `#${id1}-${id2}`;
+            }
+            return `#${ids}`;
+          }
+
+          _transformId(rootId, expansionData) {
```

Comment:
This function is a bit complicated, I think it should be documented with a comment or two